README.md

```diff
@@ -6,4 +6,4 @@ Amy/Jim/Ellesha/Charlotte robot
 - CamJam edukit or the bot Jim gave Amy, I don't remember lols
 
 # How to run this
-`python base_motor_code.py --amybot` or `python base_motor_code.py --camjambot`
\ No newline at end of file
+`python base_motor_code.py --amybot AMYBOT` or `python base_motor_code.py --camjambot CAMJAMBOT`
```

Comment:
sorry my bad on this one, can you change [these lines](https://github.com/ForestFighters/forestfighter/blob/master/base_robot_code.py#L78-L79)
```
group.add_argument("--amybot", help="Use the kit amy has (whatever Jim provided)")
group.add_argument("--camjambot", help="Use the camjam edubot kit")
```
to:
```
group.add_argument("--amybot", help="Use the kit amy has (whatever Jim provided)", action="store_true")
group.add_argument("--camjambot", help="Use the camjam edubot kit", action="store_true")
```
then this should work :)